### PR TITLE
Create generic attributes component

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it adds generic attributes component for further usage in product, page and variant details pages.
According to design and according to schema, attributes are the same objects for products, pages and variants, and also it is easier to handle them in one component.

Feature branch: https://github.com/mirumee/saleor-dashboard/pull/884

**PR intended to be tested with API branch:**  feature/file-uploads
https://github.com/mirumee/saleor/pull/6568

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Data-test are added for new elements.
1. [x] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://feature-file-uploads.api.saleor.rocks/graphql/

### CODE REVIEW

File: src/fragments/attributes.ts [hunk 1]
```diff
@@ -25,9 +34,7 @@ export const attributeDetailsFragment = gql`
     storefrontSearchPosition
     valueRequired
     values {
-      id
-      name
-      slug
+      ...AttributeValueFragment
       type
```
Comment (krzysztofwolski) [inline src/fragments/attributes.ts:38]: Shouldnt type also be here?

Comment (orzechdev) [reply src/fragments/attributes.ts]: Actually, `type` is a deprecated field, so it shouldn't be anywhere. I have made an investigation and I have found it is not even used anywhere, so I'll remove it completely.

Comment (orzechdev) [reply src/fragments/attributes.ts]: done

**Comment by patrys** [comment]
Task linked: [SALEOR-1660 Create generic component to use in product, variant and page details pages](https://app.clickup.com/t/2549495/SALEOR-1660)

**Review by mmarkusik — APPROVED** [review]

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**